changelog/2.26.0.md

```diff
@@ -52,3 +52,5 @@ Released 2025-01-14
 ### Other(s)
 
 - [#406](https://github.com/elastisys/compliantkubernetes-kubespray/pull/406) - other: Update kubespray fork for v2.26.0-ck8s2 [@Ajarmar](https://github.com/Ajarmar)
+
+#
```

Comment:
Please add changes for `## v2.26.0-ck8s3` :)

Reply:
Isn't is added by the next step in the release process?
I get confused here, I previously added it and then it was either getting duplicated or making the release fail in the next step

Reply:
Added them manually just now
Please check again @davidumea 
Thx